hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -276,13 +283,19 @@ public TransactionContext startTransaction(
       return ctxt;
     }
 
-    return TransactionContext.newBuilder()
+    TransactionContext ctxt = TransactionContext.newBuilder()
         .setClientRequest(raftClientRequest)
         .setStateMachine(this)
         .setServerRole(RaftProtos.RaftPeerRole.LEADER)
         .setLogData(raftClientRequest.getMessage().getContent())
         .setStateMachineContext(omRequest)
         .build();
+    if (LOG.isDebugEnabled()) {
+      long term = ctxt.getLogEntry() != null ? ctxt.getLogEntry().getTerm() : -1;
+      long index = ctxt.getLogEntry() != null ? ctxt.getLogEntry().getIndex() : -1;
+      LOG.debug("Start a new transaction. term = {}, index = {}", term, index);
+    }
```

Comment:
Revert this change since `startTransaction` neither has log entry nor term-index.